packages/angular_devkit/architect_cli/bin/architect.ts

```diff
@@ -78,24 +78,36 @@ async function _executeTarget(
   parentLogger: logging.Logger,
   workspace: workspaces.WorkspaceDefinition,
   root: string,
-  argv: minimist.ParsedArgs,
+  argv: yargsParser.Arguments,
   registry: schema.SchemaRegistry,
 ) {
   const architectHost = new WorkspaceNodeModulesArchitectHost(workspace, root);
   const architect = new Architect(architectHost, registry);
 
   // Split a target into its parts.
-  const targetStr = argv._.shift() || '';
+  const {
+    _: [targetStr = ''],
+    help,
+    ...options
+  } = argv;
   const [project, target, configuration] = targetStr.split(':');
   const targetSpec = { project, target, configuration };
 
-  delete argv['help'];
   const logger = new logging.Logger('jobs');
   const logs: logging.LogEntry[] = [];
   logger.subscribe((entry) => logs.push({ ...entry, message: `${entry.name}: ` + entry.message }));
 
-  const { _, ...options } = argv;
-  const run = await architect.scheduleTarget(targetSpec, options, { logger });
+  // Camelize options as yargs will return the object in kebab-case when camel casing is disabled.
+  const camelCasedOptions: json.JsonObject = {};
+  for (const [key, value] of Object.entries(options)) {
+    if (/[A-Z]/.test(key)) {
+      throw new Error(`Unknown argument ${key}. Did you mean ${strings.dasherize(key)}?`);
+    }
+
+    camelCasedOptions[strings.camelize(key)] = value;
```

Comment:
The `yargs-parser` typings don't appear to include them but the package itself exports its `camelize` and `decamelize` utility functions on the `yargsParser` object.
https://github.com/yargs/yargs-parser/blob/0fdbfa1fe960f76d520f75437e97c79727804908/lib/index.ts#L59-L60
For argument handling consistency, might be better to use them instead.